mlir/include/mlir/Dialect/SCF/IR/SCFOps.td

```diff
@@ -875,25 +875,25 @@ def ReduceOp : SCF_Op<"reduce", [
     DeclareOpInterfaceMethods<RegionBranchTerminatorOpInterface>]> {
   let summary = "reduce operation for scf.parallel";
   let description = [{
-    "scf.reduce" is the terminator for "scf.parallel" operations. It can model
+    The `scf.reduce` is the terminator for `scf.parallel` operations. It can model
```

Comment:
Nit: I think there should be no `The` here. Or add `operation`.

Reply:
Hello! Thank you, missed `operation`. Fixed.